console/src/main/scripts/plugins/metrics/ts/appServerDetails.ts

```diff
@@ -685,14 +706,14 @@ module HawkularMetrics {
           .withName('CPU usage')
           .withDescription(description)
           .withActions(this.defaultAction)
-          .withTags({ resourceId: this.feedId, resourceName: cpuResource['name'] })
+          .withTags({resourceId: this.feedId, resourceName: cpuResource['name']})
           .withContext(
-          new AlertDefinitionContextBuilder(AlertDefinitionContext.TRESHOLD_TRIGGER_TYPE)
```

Comment:
How about we create new task for refactoring these builders. Since it is not exactly the point of this issue and it might take some time to refactor all those builders.